src/views/profile/Edit.vue

```diff
@@ -31,6 +31,9 @@
         </v-card-text>
 
         <v-card-actions>
+          <v-btn color="error" flat :loading="loading" @click="cleanCache()">
+            Clean cache
+          </v-btn>
```

Comment:
Can you add a confirmation alert that tells what's gonna happen (which data is deleting) before doing it?

Reply:
being cache it does not eliminate any important data, the only feedback is that after the button's click the app is re-loaded more slowly (the first time) to download fresh data. Don't know if is important to notify this behavior to the user..